src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.WebKey.Tests/WebKeyOperationsTest.cs

```diff
@@ -21,6 +22,25 @@ public void ClearMemoryTest()
             }
         }
 
+        [Fact]
+        public void JwkEqualsTests()
```

Comment:
There should be a test for when key2 is null, which is the actual bug being fixed here.